### PR TITLE
FISH-1423 Documentation

### PR DESCRIPTION
**Pull request by kalinchan**

### CODE REVIEW

File: docs/modules/ROOT/pages/documentation/payara-server/asadmin-commands/enabling-jlink-event-expansion.adoc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+= Enable JLink Event Expansion
+
+When escaping characters in the CLI and Asadmin multimode using the \ character JLink removes the escape character for any character. This occured due to JLink updating from 2.x to 3.x, previously, the escape character was only removed for special characters.
```
Comment (rdebusscher) [inline docs/modules/ROOT/pages/documentation/payara-server/asadmin-commands/enabling-jlink-event-expansion.adoc:3]: JLine instead of JLink

Comment (rdebusscher) [reply docs/modules/ROOT/pages/documentation/payara-server/asadmin-commands/enabling-jlink-event-expansion.adoc:3]: Can you describe the link with the events as mentioned in the title?

**Review by rdebusscher — CHANGES_REQUESTED** [review]

**Review by AlanRoth — CHANGES_REQUESTED** [review]
I would also add some examples of scenarios where users may wish to enable this feature, under a 'Usage' or 'Examples' heading after the 'Enabling JLine Event Expansion'

**Comment by AlanRoth** [comment]
Also small pet peeve but I would rename the file to 'Enable JLine Event Expansion'

**Comment by kalinchan** [comment]
@AlanRoth  @rdebusscher I have made the suggested changes

File: docs/modules/ROOT/pages/documentation/payara-server/asadmin-commands/Enable-JLine-Event-Expansion.adoc [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Enable this feature if you would like the ability to use the JLine special characters in the Asadmin multimode.
+
+This is disabled by default. 
+
+== Enabling JLine Event Expansion
+Set the environment variable `+AS_ADMIN_DISABLE_EVENT_EXPANSION+` to `+false+` 
+
+=== For Linux
+. Open the Terminal
+. Tyoe the following command
```
Comment (rdebusscher) [inline docs/modules/ROOT/pages/documentation/payara-server/asadmin-commands/Enable-JLine-Event-Expansion.adoc:12]: Typo

**Review by rdebusscher — CHANGES_REQUESTED** [review]
Can you briefly describe or link to a description of what JLine Event expansion is/does.

**Review by rdebusscher — APPROVED** [review]

**Review by AlanRoth — APPROVED** [review]
LGTM

**Pull request merged**